src/main/scala/za/co/absa/commons/io/LocalFileSystemUtils.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2020 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.commons.io
+
+import java.io.File
+import java.nio.charset.StandardCharsets
+import java.nio.file.{Files, Paths}
+
+object LocalFileSystemUtils {
+
+  /**
+   * Check if a given files exists on the local file system
+   */
+  def localExists(path: String): Boolean = {
+    new File(path).exists()
+  }
+
+  /**
+   * Reads a local file fully and returns its content.
+   *
+   * @param path A path to a file.
+   * @return The file's content.
+   */
+  def readLocalFile(path: String): String = {
+    Files.readAllLines(Paths.get(path), StandardCharsets.UTF_8).toArray.mkString("\n")
+  }
```

Comment:
Should the charset be a parameter with a default value?

Reply:
I would suggest `org.apache.commons.io.FileUtils.readFileToString()` from Apache Commons IO as an existing alternative method that does exactly the same thing but directly, without pointless intermediate splitting into lines followed by concatenating everything back in one piece.

Reply:
Added charset parameter and I agree that `org.apache.commons.io.FileUtils.readFileToString()` can be used as an alternative. But given that I didn't know about this function before your comment, maybe we should keep `readLocalFile` in our commons? Here we have a smaller set of functionalities that in a way is easier to discover.

Reply:
Maybe with `@deprecated` annotation then? I think it would be useful for a person that has found this function to know that there is a better alternative?

Reply:
Makes sense, will add that.